app/src/main/java/com/github/palFinderTeam/palfinder/meetups/fragments/MeetupFilterFragment.kt

```diff
@@ -107,13 +108,18 @@ class MeetupFilterFragment(val viewModel: MapListViewModel) : DialogFragment() {
             viewModel.startTime.value?.toSimpleTime(),
             minDate = Calendar.getInstance()
         ).thenAccept {
-            val interval = viewModel.startTime.value?.timeInMillis?.let { it1 ->
+            var interval = viewModel.startTime.value?.timeInMillis?.let { it1 ->
                 viewModel.endTime.value?.timeInMillis?.minus(
                     it1
                 )
             }
+            if(interval == null){
+                interval = (BASE_HOURS_INTERVAL*3600*1000).toLong()
```

Comment:
Either put a constant or a small comment saying that it is equivalent to one day (I think) in milliseconds

Reply:
Or a constant like `BASE_HOURS_INTERVAL`?